csrc/scheduler/tools/loop_domain_scheduler.h

```diff
@@ -23,5 +24,17 @@ void scheduleLoopDomainsLike(
     const std::vector<TensorView*>& tvs,
     const std::vector<IterDomain*>& ref_loop_dom);
 
+// Replay a transform expr on the loop domain of each of the given
+// tensors. If the input of the transform matches with the loop
+// domain, the transform is replayed as a forward op. If the output
+// matches with the loop domain, it's replayed as a backward
+// op. The loop domain of each tensor is updated with the replayed
+// transform expr. If it's replayed as a forward op, the outputs
+// replace the inputs in the loop domain. If it's replayed as a
+// backward op, the inputs replace the outputs in the loop domain.
```

Comment:
Just a nitpick to clarify what "matching" means in this case:
```suggestion
// tensors. If the input of the transform is exact mapped with the loop
// domain, the transform is replayed as a forward op. If the output
// is exact mapped with the loop domain, it's replayed as a backward
// op. The loop domain of each tensor is updated with the replayed
// transform expr. If it's replayed as a forward op, the outputs
// replace the inputs in the loop domain. If it's replayed as a
// backward op, the inputs replace the outputs in the loop domain.
```

Reply:
On first read it wasn't clear that ultimately this tool is meant to allow us to schedule a sequence of tensors that includes some transforms such that all their loop domains are exact mapped. I think showing the `Resize` example from the test here will be helpful.